### PR TITLE
Update docs and add code samples

### PR DESCRIPTION
**Pull request by tsolakoua**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -270,11 +270,11 @@ List of supported endpoints
     amadeus.travel.predictions.trip_purpose.get(originLocationCode='ATH', destinationLocationCode='MAD', departureDate='2020-08-01', returnDate='2020-08-12', searchDate='2020-06-11')
 
     # Flight Delay Prediction
-    amadeus.travel.predictions.flight_delay.get(originLocationCode='BRU', destinationLocationCode='FRA', departureDate='2020-01-14', \
-    departureTime='11:05:00', arrivalDate='2020-01-14', arrivalTime='12:10:00', aircraftCode='32A', carrierCode='LH', flightNumber='1009', duration='PT1H05M')
+    amadeus.travel.predictions.flight_delay.get(originLocationCode='BRU', destinationLocationCode='FRA', departureDate='2020-11-14', \
```
Comment (anthonyroux) [inline README.rst]: Is this a real flight? Otherwise if you look in RunScope I added an example for a real flight in the future

Comment (tsolakoua) [reply README.rst]: @anthonyroux Yes, it worked in postman

Comment (anthonyroux) [reply README.rst]: On the AI side they do 0 checks you can call the APIs with random data even in the past and it always answers ;). Doesn't mean it is a real flight.

Comment (tsolakoua) [reply README.rst]: oops, I didn't know! I will update the example with the one from RunScope then.

**Pull request closed without merging**